spec/fixtures/canonical/sync/weapons.json

```diff
@@ -87,33 +87,37 @@
         "item_code": "0005AD9D",
         "quantity": 1
       }
-    ]
+    ],
+    "powers": []
   },
   {
     "attributes": {
-      "name": "Zephyr",
-      "item_code": "XX00CFB6",
+      "name": "Glass Bow of the Stag Prince",
```

Comment:
We wanted to include the weapon that's enchanted with a power and not an enchantment, to make sure that one is synced properly.